coqbot-ci.sh

```diff
@@ -98,7 +98,8 @@ printf "::group::make %s (passing)\n" "${CI_TARGET}"
 # mv "${CI_BASE_BUILD_DIR}"/coq-passing "${COQ_CI_BASE_BUILD_DIR}"
 ln -s "${CI_BASE_BUILD_DIR}"/coq-passing "${COQ_CI_BASE_BUILD_DIR}"
 pushd "${COQ_CI_BASE_BUILD_DIR}"
-make -f Makefile.ci GITLAB_CI=1 ${CI_TARGET}
+GITLAB_CI=1 dev/ci/ci-wrapper.sh ${CI_TARGET#ci-}
```

Comment:
where does CI_TARGET come from? with coq/coq#19925 we need to loop over CI_TARGETS

Reply:
It comes from the name of the job that minimization is requested on 
https://github.com/coq/bot/blob/e838d62bd395390659d4aa4805841c163420eb54/run_ci_minimization.sh#L54 which is set by matching on the job name at https://github.com/coq/bot/blob/e838d62bd395390659d4aa4805841c163420eb54/src/actions.ml#L905 and read in at https://github.com/coq-community/run-coq-bug-minimizer/blob/b6cef6ad2bc1baa1399286eac6db6f909dc8b59d/coqbot-config.sh#L50

How are we supposed to get the info about CI_TARGETS?

Reply:
I guess we can add it to the check tab (where it already says the image and dependencies)

Reply:
Could you prepare a PR to coqbot to do this, and to dump the information to `coqbot.ci-target` via https://github.com/coq/bot/blob/e838d62bd395390659d4aa4805841c163420eb54/src/actions.ml#L905 ?